SCClassLibrary/Common/Collections/Scale.sc

```diff
@@ -185,8 +190,13 @@ Tuning {
 	}
 
 	*doesNotUnderstand { |selector, args|
-		var tuning = this.newFromKey(selector, args).deepCopy;
-		^tuning ?? { super.doesNotUnderstand(selector, args) }
+	    var tuning = this.newFromKey(selector, args);
+        if(tuning.notNil) { ^tuning.deepCopy };
+        ^super.doesNotUnderstand(selector, args)
+	}
+
+	*doesNotUnderstandWithKeys {|selector, argsArray, keywordArgsAsPairs|
+        ^Tuning.performWith(\newFromKey, [selector] ++ argsArray, keywordArgsAsPairs.asEvent);
```

Comment:
better remove `asEvent` here. In`functionPerformWith`, which is the latest point of conversion, it can also be a key value pair, because there, `keysValuesDo` is called, which works for both anyway. This also saves some memory for conversions.